test/asv.linux.conf.json

```diff
@@ -33,27 +33,19 @@
     "install_command": [
         "python -c \"import shutil; shutil.rmtree('{build_dir}/qiskit', True)\"",
         "python -c \"import shutil; shutil.rmtree('{build_dir}/qiskit_aer.egg-info', True)\"",
-        "pip install git+https://github.com/Qiskit/qiskit-terra",
-        "pip install git+https://github.com/Qiskit/qiskit-aqua",
+        "pip install -U qiskit-terra",
```

Comment:
I'm wondering if we want to use `matrix` here to list qiskit-terra at a specific version so the benchmark results are tied to the qiskit-terra version (so if we bump the terra version in the matrix it shows as different from earlier runs).